### ISSUE TITLE
Getting the drone's takeoff time

### ISSUE THREAD

**Issue by jayvenn** [dji-sdk/Mobile-SDK-iOS#164]
I am wondering if there is a way to get the drone's take off time accurately even for edge cases.

At the moment, I am generating the take off time myself by listening to the flightController(_:didUpdate:) event's state.isFlying property. In addition, I have my own take off first time boolean property and take off date properties. Everything works well under a condition.

The condition is that the user will have to be connected to the app and on the drone's camera view in the app before the drone's take off.

The edge case is first taking off with the drone before entering the drone's camera view in the app. 

Basically user fly the drone then connect to the app. Then the take off time recorded will be when the user enter the App's drone camera view and not the actual take off time.  

The take off time is used everytime a new media generated event is fired. Any help on handling this edge case will be appreciated. Thanks!

**Comment by aksh1t** [comment]
Hi @nhantrivinh, I'm afraid that with the current interfaces for the SDK, it is not possible to get the takeoff time if you connect an SDK based app to the drone after takeoff.

**Comment by dji-dev** [comment]
Hi @jayvenn, if you still have more issues related to DJI SDK, please help to report them to dev@dji.com, as it's our official channel for developers to request DJI SDK support now.
